syntax = "proto2";

package recursive_dependency_embedding;

import "sequence_node.proto";

message SequenceNodeSequence {
    repeated SequenceNode trees = 1;
    optional int32 idx_correct = 2 [default = 0];
}